backends/bmv2/simpleSwitch.h

```diff
@@ -46,12 +46,12 @@ class SimpleSwitch {
  public:
     void modelError(const char* format, const IR::Node* place) const;
     void convertExternObjects(Util::JsonArray *result, const P4::ExternMethod *em,
-                              const IR::MethodCallExpression *mc, const IR::StatOrDecl *s);
+                              const IR::MethodCallExpression *mc, const IR::StatOrDecl *s, const bool& emitExterns);
```

Comment:
Did you run cpplint? These lines are too long.

Reply:
I ran. I realized that the headers files were not include in the rule for the cpplint. My next commit will include it.